recipe/meta.yaml

```diff
@@ -25,6 +27,11 @@ requirements:
   run:
     - python
 
+test:
+  commands:
+    - dir %PREFIX%\pythoncom%CONDA_PY%.dll
```

Comment:
I believe there is an `exist` like  command you can use there. Not sure though. (I suck on Windows.)